_dashboards/management/accelerate-external-data.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Optimize query performance of external data sources
+parent: Data sources
+nav_order: 30
+has_children: false
+---
+
+# Optimize query performance of external data sources
+Introduced 2.11
+{: .label .label-purple }
+
+
```

Comment:
SMEs: We need to explain why users would use a skipping index and why they would use a covering index. Please provide those use cases.

Reply:
* Feature docs: https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md
* Covered indexes -> ingest all/partial data from the source into OpenSearch and unlock full OpenSearch dashboarding and plugin functionality.
* Skipping Indices -> ingest data based on skip acceleration methods [Partition, min max, value set] to accelerate data scan by maintaining compact aggregate data structure.